components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/filter/AuthenticationEndpointFilter.java

```diff
@@ -70,6 +70,9 @@ public class AuthenticationEndpointFilter implements Filter {
     private static final String URI_PASSIVESTS_LOGIN = "passivests_login.do";
     private static final String URI_OAUTH2_LOGIN = "oauth2_login.do";
     public static final String ATTRIBUTE_SKIP_PROPERTY = "skip";
+    private static final String ORGANIZATION_AUTHENTICATOR = "OrganizationAuthenticator";
```

Comment:
Can't we use framework constants

Reply:
FrameworkConstants.Application.CONSOLE_APP

![Screenshot 2023-12-11 at 14 36 42](https://github.com/wso2/carbon-identity-framework/assets/35717390/c208163e-db43-4184-9eda-ef896a46c789)

Reply:
No. This is packed in side webapp libs. So it will add a number of additional jars.